CSC631-MultiplayerGameDev-Group-Project/TeamFourProject
# Marc environment fix

# About
This is to fix error preventing pull request. There were problems with lfs for marc's scene.
## Testing this Pull Request
1. Activate Assets/LocalWorkspace/marc-environment/envo1 scene.
2. Verify that the environment is still functional and that there is sounds for the effects.
